database/seeder.js

```diff
@@ -0,0 +1,95 @@
+const MongoClient = require('mongodb').MongoClient;
+
+const url = 'mongodb://localhost:27017';
+
+const loremIpsum = require("lorem-ipsum").loremIpsum;
+
+
+
+const generateSentences = () => {
+return  loremIpsum({

```

Comment:
Check your indentations in database/seeder.js - nested content should be seperated by an indentation of two spaces. Ex: 28-29, 45, 50-63.